lib/livebook_web/live/hooks/user_hook.ex

```diff
@@ -33,9 +33,8 @@ defmodule LivebookWeb.UserHook do
   # attributes if the socket is connected. Otherwise uses
   # `user_data` from session.
   defp build_current_user(session, socket) do
-    %{"current_user_id" => current_user_id} = session
-    user = %{User.new() | id: current_user_id}
     identity_data = Map.new(session["identity_data"], fn {k, v} -> {Atom.to_string(k), v} end)
+    user = %{User.new() | id: identity_data["id"]}
```

Comment:
The changeset respects `"id"`, shouldn't it be set by `Livebook.Users.update_user`?